x-pack/plugins/endpoint/public/applications/endpoint/store/hosts/middleware.test.ts

```diff
@@ -56,7 +56,7 @@ describe('host list middleware', () => {
     await sleep();
```

Comment:
we should remove the sleeps and instead use the new utility that allows you to wait for a given action to be dispatched - in this case, its probably the `serverReturnedHostListData` action.